### PR TITLE
Member Remove

### PR DESCRIPTION
**Pull request by mienaikoe**
- Moves invite/uninvite to TeamController
- Adds an endpoint for removing a member
- Adds an action to roster to remove a member

### CODE REVIEW

File: server/controllers/TeamController.js [hunk 1]
```diff
@@ -62,6 +78,205 @@ const TeamController = {
     next();
   },
 
+  async invitePerson(req, res, next){
```
Comment (k3ntako) [inline server/controllers/TeamController.js]: I think we should call this `invitePeople`

File: server/controllers/TeamController.js [hunk 2]
```diff
@@ -62,6 +78,205 @@ const TeamController = {
...
+          },{
+            non_attendees: person_id,
+          }
+        ]
+      });
+      
+      // Pull them from all boats they appear on
+      await Boat.updateMany({
+        event: {
+          $in: attendingEvents//.map( event => event._id )
```
Comment (k3ntako) [inline server/controllers/TeamController.js]: you have a comment here.

File: server/routes.js [hunk 3]
```diff
@@ -60,17 +60,18 @@ router.route( '/me/settings').
...
-  delete(                                requireCoach,    PersonController.uninviteFromMyTeam);
 router.route( '/people/:person_id').
   get(                                   requireVerified, PersonController.find).
   put(                                   requireCoach,    PersonController.update);
 
 router.use(   '/team',                   requireVerified);
 router.route( '/team/mine').
   get(                                                    TeamController.mine);
+router.route( '/team/invite').
+  post(                                  requireCoach,    TeamController.invitePerson).
```
Comment (k3ntako) [inline server/routes.js]: same as above, `invitePeople` instead of `invitePerson`

File: server/tests/SessionControllerTest.js [hunk 4]
```diff
@@ -106,7 +105,7 @@ describe('SessionController', function() {
         }
       }
 
-      PersonController.inviteToMyTeam({
+      TeamController.invitePerson({
```
Comment (k3ntako) [inline server/tests/SessionControllerTest.js]: same as above, `invitePeople` instead of `invitePerson`

File: server/tests/TeamControllerTest.js [hunk 5]
```diff
@@ -36,4 +37,106 @@ describe('TeamController', function() {
     })
   })
 
+
+  let invited_person_id = null;
+
+  describe('#invitePerson', function(){
```
Comment (k3ntako) [inline server/tests/TeamControllerTest.js]: same as above, `invitePeople` instead of `invitePerson`

File: server/tests/TeamControllerTest.js [hunk 6]
```diff
@@ -36,4 +37,106 @@ describe('TeamController', function() {
...
+        }
+      },{
+        json: function(response){
+          assert(response.okay);
+          Team.findById(coach_session.team_id).then( team => {
+            const foundMember = team.members.find( mpid => mpid.equals(newPerson._id) )
+            assert(!foundMember)
+            done();
+          }, done)
+        }
```
Comment (k3ntako) [inline server/tests/TeamControllerTest.js]: maybe this is too much, but ideally we would test if they were removed from events/boats/assignments.

File: server/tests/TeamControllerTest.js [hunk 7]
```diff
@@ -36,4 +37,106 @@ describe('TeamController', function() {
     })
   })
 
+
+  let invited_person_id = null;
+
+  describe('#invitePerson', function(){
+
+    it('should invite a person to my team', function(done){
+      TeamController.invitePerson({
```
Comment (k3ntako) [inline server/tests/TeamControllerTest.js]: same as above, `invitePeople` instead of `invitePerson`

**Comment by mienaikoe** [comment]
Oh damn that's a great Idea to move it to the person page.

File: server/tests/TeamControllerTest.js (hunk 6 above)
Comment (mienaikoe) [reply server/tests/TeamControllerTest.js]: good call on this. There were a few things faulty about the test.

**Comment by mienaikoe** [comment]
ready for 2nd review

File: client/pages/Person/index.css [hunk 8]
```diff
@@ -5,19 +5,14 @@
...
-.coach{
-  width: 320px;
 }
-
-.card{
-  padding: 2rem;
+.sidebar{
+  width: 240px;
+  margin-left: 2rem;
 }
```
Comment (k3ntako) [inline client/pages/Person/index.css]: You're not using this. I think you forgot to add the classname. 

Is this why the cards are super wide?

Comment (mienaikoe) [reply client/pages/Person/index.css]: Yea so I decided to move the coach sections down and get rid of the sidebar, caus there really isn't any tertiary information that's important to be above the fold on this page.

Also coach just asked if it's possible to "archive" (basically putting people under the line) so I'll see if I can bake that into this sprint.

Comment (mienaikoe) [reply client/pages/Person/index.css]: ^ to clarify, I'm not going to put that extra feature into this PR

**Pull request merged**